distro/src/config/hazelcast.yaml

```diff
@@ -0,0 +1,2 @@
+hazelcast:
+  cluster-name: dev
```

Comment:
Isn't it the default? Perhaps clusters started via CLI should have a different name?

I don't really have an opinion here. Just raising a concern to think about. It's my first time when developing a CLI

Reply:
I thought the same initially, but then MC couldn't connect to the Hz instances started by CLI by default. Maybe there is a way to add the Hz config to the MC when starting, I'll check with the MC team.

Reply:
If all options are default, do we even need this file here?

Reply:
I didn't put it initially, but users asked to see the default configuration file & location that the started Hazelcast member uses. Then we added it.